### DISCUSSION TITLE
New install using Proxmox VM

### DISCUSSION THREAD

**Discussion by 4mikestephens** [opendcim/openDCIM#1486] [General]
I have used openDCIM before, but have never set it up myself. 

FYI - I am a cable guy, Not a software/computer engineer.  I would like to get away from the spreadsheets an onto this program.

Following instructions from https://github.com/opendcim/openDCIM/wiki/Ubuntu#create-ssl-certificate   22.04

Install went ok, but;
1) I have a API Documentation Swagger Icon/Link on the bottom of the menu. Which I did not see on any other examples.

![dcim_swagger](https://github.com/opendcim/openDCIM/assets/81234321/772d9093-2fbb-4e56-8823-95448aad8628)

2) First try at bulk cabinet upload comes up with This page isn’t working with my IP address.

Any help would be appreciated.

Mike

**Comment by wilpig** [comment]
Did you follow the full install from start to finish from https://github.com/opendcim/openDCIM/wiki/Ubuntu#ubuntu-2204-installation-instructions or did you start at the ssl portion and finish from there?

Have you reviewed the bulk operations wiki page? https://github.com/opendcim/openDCIM/wiki/Bulk-Load-Operations

The swagger link is supposed to be there and was added many releases ago, you should see it on the http://demo.opendcim.org site as well.

**Reply by 4mikestephens** [reply]
Swagger I understand
Install - I started at the beginning of 22.04 step by step.
I have done bulk operations before and if I missed something the process would fail, I am getting a page not working.  I will have to look into the database and make sure I did not make an error there.

Thanks

**Reply by wilpig** [reply]
Check the apache log to see if there is an error listed there as well.  Otherwise it's likely just a data validation problem with the bulk import causing something to go haywire.

**Comment by 4mikestephens** [comment]
[error.log](https://github.com/opendcim/openDCIM/files/13619588/error.log)
 
Invalid cell coordinate @2

I hope this helps

**Reply by wilpig** [reply]
I would say that's bad data in your spreadsheet that is getting imported then.

**Comment by 4mikestephens** [comment]
Wilbur,

Sorry for the delays, I did finally get this resolved.
It was indeed bad data somewhere, so I rebuilt the bulk import template
according to the specs on the site and it works just right.  Not sure if it
was a bad download or a character that I could not see.


Thanks for your guidance.
Thanks and Regards,

Mike Stephens
Mobile: 631 312 4537
U.S. Navy Veteran
